boa/src/syntax/lexer/token.rs

```diff
@@ -108,7 +108,7 @@ pub enum TokenKind {
 
     /// A keyword.
     ///
-    /// see: [`Keyword`](../keyword/enum.Keyword.html)
+    /// see: [`Keyword`](../../ast/keyword/enum.Keyword.html)
```

Comment:
Can you check if using directly `Keyword` and `Punctuator` also fixes the links? Because IIRC, if the type is imported it should work.

Reply:
Actually now that you mention it, it seems that by default rustdoc automatically links the struct inside the enum variant. The additional "see: <Link>" is not required".
![Screenshot 2021-10-01 at 21-42-05 TokenKind in boa syntax lexer token - Rust](https://user-images.githubusercontent.com/8318878/135703805-d50cc191-e037-4405-9475-b120c6108b6f.png)

What I mean is that the green text in the above screenshot already links to the struct. Shall I remove the two explicit links that anyway seem redundant now ?

Reply:
Yeah, you can remove them